### DISCUSSION TITLE
로그인 안되는 이슈 어떻게 해결들 하시나요?

### DISCUSSION THREAD

**Discussion by almondsky** [naverz/zepeto-studio-kor#770] [4. General]
안녕하세요, 
유니티에서 월드 만들기는 완전완전 초보입니다. 현재 제페토 상의 가이드를 읽어가며 차근차근 따라해보려고 하는데,,, 간단하고도 간단한... 로그인에서 막혀버리니 머리가 지끈 거리네요..

<img width="399" alt="Screen Shot 2022-05-17 at 2 52 20 PM" src="https://user-images.githubusercontent.com/105544091/168739009-eea671fc-8e7b-4c8b-b751-be9a121a5033.png">

제페토 가이드 상 기본 예제를 ZEPETO 앱(모바일)에서 테스트하려면, [▶︎(play)] 버튼 우측의 ZEPETO 버튼을 클릭합니다 라고 나와있어서 따라하니, 로그인이 필요하다고 해서 (저는 이메일이나 휴대폰으로 연동되어있는게 아니라, 타 플랫폼로 연동되어있어 제페토에 접속을 하기 때문에.. 우선 그 쪽 아이디, 비번을 입력했는
<img width="995" alt="Screen Shot 2022-05-17 at 2 59 48 PM" src="https://user-images.githubusercontent.com/105544091/168739610-efee8fa3-e2fa-491e-9bb9-62d1f28ece11.png">
이렇게 뜨길래, 제가 생각나는 아이디와 비번은 이것저것 다 입력해봤는데도 위와 같이 동일하게 오류가 뜨고 아예 접속이 안되네요.. 
저처럼 타 플랫폼으로 연동하신 분들은 로그인 어떻게 해서 들어가시나요..? ㅠㅠ  
<img width="589" alt="Screen Shot 2022-05-17 at 3 02 48 PM" src="https://user-images.githubusercontent.com/105544091/168740103-49d75fab-46a0-4753-8c58-e85d3d28f349.png">

아.. 그리구 로그인 없이 혹시나 해서 제페토 버튼만 꾸욱 눌러보면 아무것도 안뜨고 무지개만 계속 돌아가며 로딩중이고,, 이렇게 뜹니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 앱에서 전화번호나 이메일을 연동하신 후 진행하셔야 합니다.
